TROUBLESHOOTING.md

```diff
@@ -23,6 +23,30 @@ sqlite3.OperationalError: database is locked
 ```
 Just stop the running instance of Vectory and try again.
 
+
+### Index already exists
+
+If adding an index to the database fails in a way we haven't anticipated, it might happen that the index is loaded in Elasticsearch but not in the database. This can be checked by comparing the indices on both places:
+
+```python
+from vectory.db.models import ElasticSearchIndexModel, Query
+from vectory.es.client import ElasticKNNClient
+
+es = ElasticKNNClient()
+es_indices = es.list_indices()
+
+db_indices = Query(ElasticSearchIndexModel).get()
+```
+
+`es_indices` and `db_indices` should be the same. If they aren't, you can delete the index from Elasticsearch and add it again to the database.
```

Comment:
Although is less likely to happen, the problem could also be that the index is loaded in the db but not in es. I think we should also explain how to fix this problem.

Reply:
I don't think that this case would happen, but I agree with adding it just in case